src/icrc1_auction_api_new.mo

```diff
@@ -1073,26 +1029,22 @@ actor class Icrc1AuctionAPI(quoteLedger_ : ?Principal, adminPrincipal_ : ?Princi
     };
   };
 
-  // each 2 minutes
-  let AUCTION_INTERVAL_SECONDS : Nat64 = 120;
-  var nextSessionTimestamp = 0;
+  // A timer for running auction
+  ignore (
```

Comment:
How does this work? You want to pipe the function into `_`, right? If we use `ignore` then don't we pipe a value of `()`?

Reply:
`ignore` is written outside of the parentheses, so the function goes into the pipe, but the final return value, returned by `Timer.setTimer` is ignored